### DISCUSSION TITLE
<ayuda

### DISCUSSION THREAD

**Discussion by marRosario** [community/community#114433] [Repositories]
### Select Topic Area

Question

### Body

tengo un problema : al subir el proyecto , en el repositorio se ven todos los archivos pero cuando obtengo la URL me muestra solo dos paginas y la tercera me tira error 404 , Pueden decirme que significa eso ? Me muestra lo siguiente :  404
Archivo no encontrado

El sitio configurado en esta dirección no contiene el archivo solicitado.

Si este es su sitio, asegúrese de que el nombre del archivo coincida con la URL y con los permisos del archivo.
Para las URL raíz (como http://example.com/), debe proporcionar un index.htmlarchivo.

**Comment by Akash1134** [comment]
Hi @marRosario , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
